OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -326,31 +326,26 @@ byte[] DoublePixelData(byte[] data, Size size)
 		public IHardwareCursor CreateHardwareCursor(string name, Size size, byte[] data, int2 hotspot, bool pixelDouble)
 		{
 			VerifyThreadAffinity();
-			try
```

Comment:
We should keep this try/catch for any legitimate non-windows exceptions (e.g. size not matching data). The except branch should log the exception and return null.